src/access_control.py

```diff
@@ -63,7 +63,7 @@ def admin_role_query(self, username, group, session):
 
         # query permissions from group roles
         group_roles_query = query.join(Role.groups_collection) \
-            .filter(Group.name == group)
+            .filter(Group.name.in_(groups))
```

Comment:
Does this work if groups is `None`?

Reply:
If groups is `None` this will not work. Normally this case should not happen but I modified it to secure the code.

Reply:
Thanks - for instance  qwc-db-auth won't set `groups` in the token [1]

[1] https://github.com/qwc-services/qwc-db-auth/blob/master/src/db_auth.py#L809